cmd/podman/main_remote.go

```diff
@@ -18,6 +18,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.ConnectionName, "connection", "", "remote connection name")
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.RemoteConfigFilePath, "remote-config-path", "", "alternate path for configuration file")
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.RemoteUserName, "username", username, "username on the remote host")
+	rootCmd.PersistentFlags().IntVar(&MainGlobalOpts.Port, "port", 22, "port on remote host")
```

Comment:
Worth doing any validation on this? Making sure they gave us a positive integer between 0 and 65536, exclusive?

Reply:
yeah I like that.